src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -126,8 +127,8 @@ def login(self,
 
         credential=None
         auth_profile=None
-        authority = self.cli_ctx.cloud.endpoints.active_directory.replace('https://', '')
-        identity = Identity(authority, tenant, cred_cache=self._adal_cache)
+        adal_cache = ADALCredentialCache(cli_ctx=self.cli_ctx)
```

Comment:
>adal_cache = ADALCredentialCache(cli_ctx=self.cli_ctx) [](start = 8, length = 54)

not used?

Reply:
No idea how this line get inserted. Removed.